client/src/main/java/starting/StartBoard.java

```diff
@@ -2,31 +2,49 @@
 
 import javafx.application.Application;
 import javafx.fxml.FXMLLoader;
+import javafx.scene.Group;
 import javafx.scene.Scene;
+import javafx.scene.control.Button;
 import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
 import javafx.stage.Stage;
 
 public class StartBoard extends Application {
+
     private final double SCENE_WIDTH = 800;
     private final double SCENE_HEIGHT = 600;
     private final String START_BOARD_URL = "/fxmls/startBoard.fxml";
-
+    private final String PLAY_BOARD_URL = "/fxmls/playBoardEmpty.fxml";
 
     @Override
     public void start(Stage stage) throws Exception {
 
         stage.setResizable(false);
 
-        FXMLLoader loader = new FXMLLoader(getClass().getResource(START_BOARD_URL));
-        AnchorPane startBoard = loader.load();
+        Group startRoot = new Group();
+        Group playRoot = new Group();
+
+        Scene startScene = new Scene(startRoot, SCENE_WIDTH, SCENE_HEIGHT);
+        Scene playScene = new Scene(playRoot, SCENE_WIDTH, SCENE_HEIGHT);
 
-        StartBoardController startBoardController = loader.getController();
-        System.out.println(startBoardController.ipNumber.getText()); //TODO: change to Logger
+        FXMLLoader startLoader = new FXMLLoader(getClass().getResource(START_BOARD_URL));
+        AnchorPane startBoard = startLoader.load();
+        addNextButtonToStartBoard(stage, playScene, startBoard);
+        startRoot.getChildren().addAll(startBoard);
+
+        FXMLLoader playLoader = new FXMLLoader(getClass().getResource(PLAY_BOARD_URL));
+        AnchorPane playBoard = playLoader.load(); //TODO to uruchamia drugi wątek!!!!
```

Comment:
TODO why in polish?

Reply:
Because it was 2 o'clock in the middle of the night